homeassistant/components/light/yeelight.py

```diff
@@ -286,13 +328,428 @@ def set_flash(self, flash) -> None:
             except BulbException as ex:
                 _LOGGER.error("Unable to set flash: %s", ex)
 
+    @_cmd
+    def set_effect(self, effect) -> None:
+        """Activate effect."""
+        if effect:
+            from yeelight import (RGBTransition, TemperatureTransition,
+                                  SleepTransition, Flow, BulbException)
+            transition = int(self.config[CONF_TRANSITION])
+            if effect == EFFECT_ALARM:
```

Comment:
This is being discussed here: https://gitlab.com/stavros/python-yeelight/issues/11 so I'd propose closing this PR for now and reopening/creating a new one after the implementation hits the upstream :-)